src/compiler/sys.ts

```diff
@@ -1125,6 +1125,26 @@ namespace ts {
         return parseInt(version.substring(1, dot));
     }
 
+    function wrapStacktraceReduction(mod: any): any {
+        const result = {} as any;
+        for (const key in mod) {
+            if (!(mod[key] instanceof Function)) continue;
+            result[key] = (...args: any[]) => {

```

Comment:
Worth understanding whether an allocation for this array in each call is a potential problem.